app/src/main/java/com/github/se/studybuddies/ui/groups/GroupMembers.kt

```diff
@@ -0,0 +1,281 @@
+package com.github.se.studybuddies.ui.groups
+
+//noinspection UsingMaterialAndMaterial3Libraries
+//noinspection UsingMaterialAndMaterial3Libraries
```

Comment:
Instead of needing to add this, you should use material3 library for dropDownMenuItem and for IconButton